spotlight-api/README.md

```diff
@@ -44,6 +44,8 @@ Expected backend environment variables include:
 
 - `GOOGLE_APPLICATION_CREDENTIALS` - a relative path pointing to the JSON file containing the credentials of the service account used to communicate with Google Cloud Storage, for metric retrieval.
 - `METRIC_BUCKET` - the name of the Google Cloud Storage bucket where the metrics reside.
+- `AUTH_ENABLED` - whether or not we should require authentication to access our endpoints. Currently only used in staging to make the entire site private. No need to enable this locally unless you are developing or testing something auth-related. If set to `true` then `AUTH0_APP_METADATA_KEY` **must** be set to a supported value.
```

Comment:
This means we'll need to change the staging environment to make sure these keys are set

Reply:
the backend is in App Engine, there are GAE config files in a shared 1password document that we have to update and then store locally (untracked) in order to deploy.

Reply:
@colincadams do you know how to set the staging environment variables?

Reply:
@terryttsai @macfarlandian so is it sufficient to update gae-staging.yaml in public-dashboard-untracked-config-files.zip in 1pass? How do those get pulled in during deploy?

@terryttsai you don't have access to 1pass, right? So I can do this for you

Reply:
I don't think so, I don't see that file in 1pass

Reply:
that zip file is the "master copy" but then you have to download it and copy those files into their respective locations in your working directory. (The deployed backend does not actually use the `.env` file, which is a little confusing, it uses the environment config in the yaml for that environment. The `.env` is just for development.)

Reply:
I see. Are spotlight deploys just done manually by whomever wants to then? By running `gcloud app deploy gae-staging.yaml --project [staging_project_id]`? Or is there a more formal process?

Reply:
nope that's as formal as it gets

Reply:
Cool, ok this all makes sense, thanks!